enlighten/assets/uic_qrc/enlighten_layout.ui

```diff
@@ -15066,7 +15066,7 @@ p, li { white-space: pre-wrap; }
                                <string>whether laser power should be controlled through mW or percentage</string>
                               </property>
                               <property name="whatsThis">
-                               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;In general, all spectrometers with multi-mode lasers (&lt;bgt;MML&lt;/b&gt;) receive laser power calibrations, and are therefore recommended for control through mW. Spectrometers with single-mode lasers (&lt;b&gt;SML&lt;/b&gt;) typically do not receive laser power calibrations, and are therefore recommended for control through pulse-modulation width (&lt;b&gt;PWM&lt;/b&gt;) duty-cycle, i.e. a rough percentage of full power.&lt;/p&gt;&lt;p&gt;Therefore, most MML and SML spectrometers will not see this option, and will not be given the option to configure how laser power is controlled.&lt;/p&gt;&lt;p&gt;The special case is for Integrated Laser with Interchangeable Coupling (&lt;b&gt;ILC&lt;/b&gt;) models, which have an MML exposed through a fiber and probe.  Although these units will receive a laser power calibration in the factory, and therefore are able to set power in mW, the user may elect to use a different fiber and probe than was used to generate the calibration.&lt;/p&gt;&lt;p&gt;In such cases the calibration, although present, is invalid, and we need to give the user the ability to control the laser through percentage as well.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+                               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;In general, all spectrometers with multi-mode lasers (&lt;b&gt;MML&lt;/b&gt;) receive laser power calibrations, and are therefore recommended for control through mW. Spectrometers with single-mode lasers (&lt;b&gt;SML&lt;/b&gt;) typically do not receive laser power calibrations, and are therefore recommended for control through pulse-modulation width (&lt;b&gt;PWM&lt;/b&gt;) duty-cycle, i.e. a rough percentage of full power.&lt;/p&gt;&lt;p&gt;Therefore, most MML and SML spectrometers will not see this option, and will not be given the option to configure how laser power is controlled.&lt;/p&gt;&lt;p&gt;The special case is for Integrated Laser with Interchangeable Coupling (&lt;b&gt;ILC&lt;/b&gt;) models, which have an MML exposed through a fiber and probe.  Although these units will receive a laser power calibration in the factory, and therefore are able to set power in mW, the user may elect to use a different fiber and probe than was used to generate the calibration.&lt;/p&gt;&lt;p&gt;In such cases the calibration, although present, is invalid, and we need to give the user the ability to control the laser through percentage as well.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>

```

Comment:
Single character change `all spectometers with multi-mode lasers (&lt;b& <---`

Reply:
before: `(&lt;bgt;MML&lt;/b&gt;)`
rendering: `(<bgt;MML</b>)` incorrect XML

after: `(&lt;b&gt;MML&lt;/b&gt;)`
rendering: `(<b>MML</b>)` correct XML

OK this change is good, just needed some writing space to understand it